### ISSUE TITLE
Error in unconditional_lyrics_training.ipynb

### ISSUE THREAD

**Issue by shirishr** [warmspringwinds/pytorch-rnn-sequence-generation-classification#2]
Second cell has

rnn = RNN_classification(input_size=len(all_characters) + 1, hidden_size=512, num_classes=len(all_characters))

but **RNN_classification** is undefined

**Comment by shirishr** [comment]
I suspect **RNN_classification** is a typo.
I changed it to **RNN**.

I also made following changes for python3.6:

**Changed the code block:**

def string_to_labels(character_string):
 
    return map(lambda character: character_to_label(character), character_string)

**to**

def string_to_labels(character_string):
    
    return list(map(lambda character: character_to_label(character), character_string))


**Changed the code block:**

clip = 1.0
epochs_number = 1000

for epoch_number in xrange(epochs_number):

    for batch in trainset_loader:

        post_processed_batch_tuple = post_process_sequence_batch(batch)

**to**

clip = 1.0
epochs_number = 1000
**my_iter = iter(trainset_loader)**
for epoch_number in **range**(epochs_number):

    for count in range(len(my_iter)):

        batch = next(my_iter)
        post_processed_batch_tuple = post_process_sequence_batch(batch)


**Changed the line:**

logits, _ = rnn(input_sequences_batch_var, sequences_lengths)

**to**

logits, _ = rnn(input_sequences_batch_var, **list(sequences_lengths)**)
